viz_scripts/generic_metrics.ipynb

```diff
@@ -119,193 +119,366 @@
   },
   {
    "cell_type": "markdown",
-   "id": "modified-skiing",
+   "id": "6d44c87e",
```

Comment:
does this have to change?

Reply:
I understand the `id` have been changes because of the addition of new `markdown` and `code` blocks in the notebook.